device/bgp/bgpv4srtepolicy.yaml

```diff
@@ -0,0 +1,80 @@
+components:
+  schemas:
+    Bgp.V4Policy:
+      x-include:
+      - '../../common/common.yaml#/components/schemas/Named.Object'
+      - '../device.yaml#/components/schemas/Device.Active'
+      description: |
+        Configuration for BGP basic Segment Routing Traffic Engineering(SRTE) 
+        policy settings.
+      type: object
+      required: [ipv4_endpoint]
+      properties:
+        distinguisher:
+          description: >-
+            Identifies the SR policy in the context of <Color, Endpoint> tuple. 
+            It is used by the SR Policy originator to make unique multiple 
+            occurrences of the same SR Policy as well as candidate paths of
+            different SR Policies.
+          type: integer
+          default: 1
+        color:
+          description: >-
+            Policy color is used to match the color of the destination
+            prefixes to steer traffic into the SR Policy.
+          type: integer
+          default: 100
+        ipv4_endpoint:
+          description: >-
+            Specifies a single node or a set of nodes (e.g., an anycast
+            address). It is selected on the basis of the SR Policy type (AFI).
+          type: string
+          format: ipv4
+        next_hop_mode:
+          description: >-
+            Specify the NextHop in MP REACH NLRI. The mode for setting the IP address 
+            of the NextHop in the MP REACH NLRI can be one of the following:
+              Local IP: Automatically fills the Nexthop with the Local IP of the BGP peer.
+              If BGP peer is of type IPv6, Nexthop Encoding capability should be enabled.
+              Manual: Override the Nexthop with any arbitrary IPv4/IPv6 address.   
+          type: string
+          enum: [local_ip, manual]
+          default: local_ip
+        next_hop_address_type:
+          description: >-
+            If the Nexthop Mode is Manual, it sets the type of the NextHop IP address. 
+          type: string
+          enum: [ipv4, ipv6]
+          default: ipv4
+        next_hop_ipv4_address:
+          description: >-
+            The IPv4 address of the next hop if the Nexthop Mode is manual and
+            the Nexthop type is IPv4. If BGP peer is of type IPv6, Nexthop Encoding
+            capability should be enabled.
+          type: string
+          format: ipv4
+        next_hop_ipv6_address:
+          description: >-
+            The IPv6 address of the next hop if the Nexthop Mode is manual and
```

Comment:
For IPv6 next hop with manual Nexthop mode, this field gives the next hop IPv6 address.